dynamic-config/config-nacos/src/main/java/org/apache/servicecomb/config/nacos/archaius/sources/NacosConfigurationSourceImpl.java

```diff
@@ -41,96 +41,96 @@
 import com.netflix.config.WatchedUpdateResult;
 
 public class NacosConfigurationSourceImpl implements ConfigCenterConfigurationSource {
-  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigCenterConfigurationSource.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigCenterConfigurationSource.class);
```

Comment:
This project using google code style, maybe you need to add it to your editor. For IDEA and eclipse, this file is in $java-chassis/etc folder.

Reply:
Seems travis CI got some problem. You can first fix code style problem and submit a commit direct to this PR, the CI will be retriggerd .